docs-src/tut-7-rpc.scrbl

```diff
@@ -340,11 +230,22 @@ Alice went from 10 to 4.9999
   Bob went from 10 to 14.9999
 }
 
+@margin-note{
+What is this @tt{"venv"} thing?
+
+A Python
+@link["https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment"]{venv}
+is a "virtual environment" that sandboxes dependencies to avoid cluttering your
+system directories.
```

Comment:
This is a nice summary. Does python3 always have `venv`?

Reply:
The real answer is "no" since `venv` only became available in Python 3.3, but for all practical purposes (hopefully ours!) the answer is "yes" due to Python's EOL schedule:
https://endoflife.date/python
https://docs.python.org/3/library/venv.html

Someone would have to be using a pretty ancient version for `venv` not to be included.